doc/source/tutorial/optimize.rst

```diff
@@ -337,11 +337,73 @@ Hessian product example:
     >>> res.x
     array([1., 1., 1., 1., 1.])
 
+Trust-Region Truncated Generalized Lanczos / Conjugate Gradient Algorithm (``method='trust-krylov'``)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Similar to the ``trust-ncg`` method, the ``trust-krylov`` method is a method
+suitable for large-scale problems as it uses the hessian only as linear
+operator by means of matrix-vector products.
+In contrast to the ``trust-ncg`` method, it solves the quadratic subproblem
```

Comment:
Thanks! My original sentence wasn't implying this as I wrote "...more accurately...", but your suggestion is shorter and easier, so I took it.